LocationService/src/main/java/com/revature/caliber/location/LocationServiceApplication.java

```diff
@@ -0,0 +1,21 @@
+package com.revature.caliber.location;
+
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
+
+import com.revature.caliber.controllers.LocationController;
+
+@SpringBootApplication
+@EnableDiscoveryClient
+public class LocationServiceApplication implements CommandLineRunner {
+
+	public static void main(String[] args) {
+		SpringApplication.run(LocationServiceApplication.class, args);
+		
+	}
+	
+	@Override
```

Comment:
remove CommandLineRunner if you are not using it

Reply:
Removed in latest commit.